test/manual/amp-base-carousel/basic.amp.html

```diff
@@ -6,7 +6,7 @@
   <link rel="canonical" href="amps.html" >
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
-  <script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-0.1.js"></script>
+  <script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-1.0.js"></script>
```

Comment:
The 0.1 might be needed for some particular tests, especially since 0.1 is still the PROD version. Can we test via storybook? I like those a lot better now.

Reply:
err, yeah I didn't mean to include this. will revert.
Do you even use storybook to test bento components in amp mode?